docs/products/opensearch/howto/connect-with-curl.rst

```diff
@@ -0,0 +1,131 @@
+Use Aiven for OpenSearch with cURL
+==================================
+
+Connect to your Aiven for OpenSearch service with `cURL <https://curl.se/>`_.
+
+Variables
+---------
+
+These are the placeholders you will need to replace in the code samples:
+
+==================      =============================================================
+Variable                Description
+==================      =============================================================
+``OPENSEARCH_URI``      Service URI, including username and password, available from
+                        the service overview page
+==================      =============================================================
+
+Connect to OpenSearch
+---------------------
+
+Connect to your service with::
+
+    curl OPENSEARCH_URI
+
+If the connection is successful, one of the nodes in your cluster will respond with some information including:
+
+* ``name`` The node name you are connected to (this will be the name of your service, with the node identifier suffix)
+
+* ``version`` The version information includes both the ``distribution`` and the ``version`` of the distribution that is running
+
+Manage indexes
```

Comment:
I wonder if we should not separate out all the tasks that are listed below and include these in a different section (still in the Code samples?), so that this topic stays on the connection.

Reply:
True, "Manage indexes" and "Search or retrieve data" could be split into separate tasks ;)

Reply:
Minor, would be better to use `indices` in a plural form, usually preferred way, see please references in the official documentation (https://github.com/opensearch-project/index-management)

Reply:
Updated the indices comment. I think I'll keep this small number of basic examples together, but rename the page to make it clearer what is here.